x11-base/xcb-proto/xcb-proto-1.14-r1.ebuild

```diff
@@ -17,26 +17,36 @@ EGIT_REPO_URI="https://gitlab.freedesktop.org/xorg/proto/xcbproto.git"
 
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~ppc-aix ~x64-cygwin ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
 IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
 DEPEND=""
-RDEPEND="${PYTHON_DEPS}"
-BDEPEND="dev-libs/libxml2"
-
-REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+RDEPEND="
+	${PYTHON_DEPS}
+"
+BDEPEND="
+	${PYTHON_DEPS}
+	dev-libs/libxml2
+"
+
+ECONF_SOURCE="${S}"
```

Comment:
AFAICT this is the default value. Is it necessary?

Reply:
Not with plain `multilib-minimal`. `xorg-3` defines this but we're overriding it.

```bash
multilib_src_configure() {
        ECONF_SOURCE="${S}" econf "${econfargs[@]}"
}
```

Reply:
Thanks for the pointer and sorry for the noise.